honeybee_grasshopper_energy/src/HB Window Construction.py

```diff
@@ -0,0 +1,84 @@
+# Honeybee: A Plugin for Environmental Analysis (GPL)
+# This file is part of Honeybee.
+#
+# Copyright (c) 2019, Ladybug Tools.
+# You should have received a copy of the GNU General Public License
+# along with Honeybee; If not, see <http://www.gnu.org/licenses/>.
+# 
+# @license GPL-3.0+ <http://spdx.org/licenses/GPL-3.0+>
+
+"""
+Create an EnergyPlus window construction. Layer inputs can be either the name of
+a material within the material library or a custom material made with any of the
+EnergyPlus Material components.
+_
+Note that _layer_1 is always the outermost (exterior) layer and the last layer
+in the component is always the innermost (interior) layer.
+_
+To add more layers in the construction, zoom into the component and hit the
+lowest "+" sign that shows up on the input side. 
+_
+To remove layers from the construction, zoom into the component and hit the
+lowest "-" sign that shows up on the input side.
+-
+
+    Args:
+        _name: A unique name for the window construction.
+        _layer_1: The first outer-most material layer of the construction.
```

Comment:
Same comment about the layers but let's discuss it on the opaque construction component.

Reply:
Sounds good. See my current response.